docs/getting-started/libnetwork/CalicoSwarm.md

```diff
@@ -0,0 +1,83 @@
+    -initial-advertise-peer-urls http://$MANAGER_IP:2380 \
+    -listen-peer-urls http://0.0.0.0:2380  \
+    -initial-cluster-token etcd-cluster-1 \
+    -initial-cluster etcd0=http://$MANAGER_IP:2380 \
+    -initial-cluster-state new &
+    
+For this example, we'll only configure a single node etcd cluster on our master node. However, in a production environment, a minimum size of three nodes is recommended and can run anywhere that is accessible to the cluster.
+
+## Configure Calico
+
+Now that etcd is running, we can run Calico.
+
+To make things simpler, let's store some commonly used values as environment variables on each **Swarm node**. We will use these variables throughout the guide.
+
+    $ export MANAGER_IP=<Manager's IP Address>
+    $ export NODE_IP=<This Node's IP Address>
+    
+Now, start the Calico node service on each node in the cluster. 
+
+    $ sudo ETCD_AUTHORITY=$MANAGER_IP:2379 calicoctl node --libnetwork
+
+## Configure Swarm Cluster
+
+Now that Calico networking is configured on our cluster, lets join our nodes into a Docker Swarm cluster.
+
+We'll use the token based discovery backend, so let's first create the token which will identify our swarm. To do this, run the following on your client. Note that docker commands may need to be run with root privileges depending on your docker installation.
+
+    $ docker pull swarm
+    $ docker run --rm swarm create
+    
+The second command will print a unique cluster ID token which we must use to configure our Swarm. We'll reference this value in the tutorial as `<swarm_token>`. Your token should look something like this:
+    d435dc104abdd89af24ae2392c7338ce
+
+## Configure Swarm Nodes
+
+Before we configure our Swarm, we will need to start the Docker daemon and point the cluster-store to our running instance of etcd. 
+
+    $ sudo docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock --cluster-store=etcd://$MANAGER_IP:2379 &
+
+Now that we've got a token and a properly running daemon, we can begin to configure our Swarm. First, let's join each of our Swarm nodes to the cluster.
+
+Run the following commands on each node, replacing `<swarm_token>` with your given token.
+    
+    $ docker run -d swarm join --addr=$NODE_IP:2375 token://<swarm_token>
+    
+## Configure Swarm Manager
+
+Let's now configure the Swarm manager. Note that `<swarm_port>` in the following command can be any unused TCP port on the manager server. This is the port the client will use to communicate with the Swarm manager daemon. 
+
+    $ docker run -d -p <swarm_port>:2375 swarm manage token://<swarm_token>
+    
+## Testing Your Docker Swarm
+    
+Now we are ready to start testing our Docker Swarm. You can follow our [Demonstration guide](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/libnetwork/Demonstration.md#creating-networked-endpoints) for instructions on creating networked endpoints that utilize Calico networking.
```

Comment:
As discussed.  This is the most important part of the document, so this needs spelling out - in particular to utilize swarm to handle the multiple hosts from a single host.